db/migrate/20241023004427_backfill_note_subscriptions.rb

```diff
@@ -0,0 +1,14 @@
+class BackfillNoteSubscriptions < ActiveRecord::Migration[7.2]
+  class NoteComment < ApplicationRecord; end
+  class NoteSubscription < ApplicationRecord; end
+
+  disable_ddl_transaction!
+
+  def up
+    NoteComment.in_batches(:of => 1000) do |note_comments|
+      rows = note_comments.distinct.where.not(:author_id => nil).pluck(:author_id, :note_id)
+      attrs = %w[user_id note_id]
```

Comment:
There's no need to initialise this for every batch, it could go before the loop?

Reply:
Moved `attrs = ...` outside, but I don't expect any difference about this compared to db reads/writes.